practices/Luis Guardia/python01_practice1.py

```diff
@@ -0,0 +1,107 @@
+# excersice 1
+
+def add(number_one , number_two):
```

Comment:
you should follow this format:
add(variable_1, variable_2)

it is not necessary to add space before the comma

Please check the other methods